modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/asset/model/KBArticleAssetRendererFactory.java

```diff
@@ -151,6 +151,15 @@ private KBArticle _getKBArticle(long classPK, int status)
 			return kbArticle;
 		}
 
+		if (status == WorkflowConstants.STATUS_APPROVED) {
```

Comment:
I feel odd the "if is approved find the expired"

what do you think removing the if:
`	if (status == WorkflowConstants.STATUS_APPROVED) {`

Reply:
You are right. It looked strange. I made a little refactoring and now it looks better, I hope. See my last commit.

Reply:
👍🏻